write_buffer/src/config.rs

```diff
@@ -42,17 +51,29 @@ impl WriteBufferConfigFactory {
     /// # Panics
     /// When mock with identical name is already registered.
     pub fn register_mock(&mut self, name: String, state: MockBufferSharedState) {
+        self.set_mock(name, Mock::Normal(state));
+    }
+
+    /// Registers new mock that always fail.
+    ///
+    /// # Panics
+    /// When mock with identical name is already registered.
+    pub fn register_alway_fail_mock(&mut self, name: String) {
```

Comment:
There is a typo here:

```suggestion
    pub fn register_always_fail_mock(&mut self, name: String) {
```

Reply:
fixed